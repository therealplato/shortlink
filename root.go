package main

var root = []byte(`<!DOCTYPE html>
<html>
<head><title>short.link</title>
<style>
	body, input{
		max-width:80em;
		margin: auto;
		display: block;
	}
	input[type="text"] {
		width: 100%;
		margin: 5em auto 1em auto;
	}
	input[type="submit"] {
		width: 20%;
    height: 4em;
    font-size: 1.5em;
	}
</style>
</head>
<body>
	<form action="/create/">
		<input type="text" name="destination" placeholder="https://github.com/therealplato/shortlink/issues">
		<input type="submit" value="Shorten">
	</form>
</body>
</html>`)
