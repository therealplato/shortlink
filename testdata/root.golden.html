<!DOCTYPE html>
<html>
<head><title>short.link</title>
<style>
	body, input{
		max-width:80em;
		margin: auto;
		display: block;
	}
	input[type="text"] {
		width: 100%;
		margin: 5em auto 1em auto;
	}
	input[type="submit"] {
		width: 20%;
    height: 4em;
    font-size: 1.5em;
	}
</style>
</head>
<body>
	<form action="create" method="post">
		<input type="text" name="slug" placeholder="slug (leave blank for random)">
		<input type="text" name="link" placeholder="https://github.com/therealplato/shortlink/issues">
		<input type="submit" value="Shorten">
	</form>
</body>
</html>